### DISCUSSION TITLE
Using  deck.gl with Google bas maps while using React

### DISCUSSION THREAD

**Discussion by omar-3** [visgl/deck.gl#5408] [Q&A]
I tried to mimc the [pure-js](https://github.com/visgl/deck.gl/blob/8.3-release/examples/get-started/pure-js/google-maps/app.js) integration with google base maps in react project but I couldn't.

Anyone could help me out?

**Comment by sclebal** [comment]
Hi @omar-3, for integrating deck.gl + Google Maps you can use [@carto/react-basemaps](https://www.npmjs.com/package/@carto/react-basemaps) package. You can check the reference [right here](https://github.com/CartoDB/carto-react/blob/master/packages/react-basemaps/reference.md).
